mmv1/third_party/terraform/services/appengine/resource_app_engine_flexible_app_version_test.go

```diff
@@ -166,6 +182,21 @@ resource "google_project" "my_project" {
   billing_account = "%{billing_account}"
 }
 
+resource "google_compute_network" "network" {
```

Comment:
We'll need to use a `google_project_service` to enable `compute.googleapis.com` in the project & depend on that on this resource (in both configs)

Reply:
Done